kornia/augmentation/_2d/mix/cutmix.py

```diff
@@ -133,3 +133,124 @@ def apply_transform(  # type: ignore
             )
 
         return out_inputs, torch.stack(out_labels, dim=0)
+
+
+class RandomCutMixV2(MixAugmentationBaseV2):
+    r"""Apply CutMix augmentation to a batch of tensor images.
+
+    .. image:: _static/img/RandomCutMix.png
+
+    Implementation for `CutMix: Regularization Strategy to Train Strong Classifiers with
+    Localizable Features` :cite:`yun2019cutmix`.
+
+    The function returns (inputs, labels), in which the inputs is the tensor that contains the mixup images
+    while the labels is a :math:`(\text{num_mixes}, B, 3)` tensor that contains (label_permuted_batch, lambda)
+    for each cutmix.
+
+    The implementation referred to the following repository: `https://github.com/clovaai/CutMix-PyTorch
+    <https://github.com/clovaai/CutMix-PyTorch>`_.
+
+    Args:
+        height: the width of the input image.
+        width: the width of the input image.
+        p: probability for applying an augmentation to a batch. This param controls the augmentation
+                   probabilities batch-wisely.
+        num_mix: cut mix times. Default is 1.
```

Comment:
```suggestion
        num_mix: cut mix times.
```
defaults are also inlcuded